### ISSUE TITLE
Xbox One Controller PC support

### ISSUE THREAD

**Issue by SupaFlyIs187** [nesbox/emulator#74]
My TV died so I switched from Xbox One to PC and using my Xbox One controller on my pc with the Nesbox app, virtually every rom is unplayable as the "B" button (default settings in which B is assigned as B) backs me out of the rom to the menu to choose a game, where 99% of games call for use of B. I tried to assign Left bumper as B but it will not save.

**Comment by testcooldude4444** [comment]
try hooking up your xbox one with your monitor sure it's small but playable all you need is a HDMI cord which it should have came with but if you have a laptop trying find a way for the HDMI ( If they are is one) to be able to input the xbox one connection try finding it online.

**Comment by SupaFlyIs187** [comment]
Yeah I bought a new tv and I have all the roms on my one drive for the XB1.

**Comment by hughes11** [comment]
I also have had this issue b Just backs you out
